client/src/ui/BitbakeTerminal.ts

```diff
@@ -119,6 +120,7 @@ export class BitbakePseudoTerminal implements vscode.Pseudoterminal {
     processResolved.stdout?.on('data', (data) => {
       this.output(data.toString())
       logger.debug(data.toString())
+      this.outputDataString += data.toString()
```

Comment:
note: this is suboptimal for most of the calls to BitbakeTerminal. You could skip copying the output buffers unless `this.isTaskTerminal()`.

Reply:
Done.